sla_per_user_system_test.py

```diff
@@ -779,3 +814,56 @@ def get_test_status(self) -> str:
             except KeyError as exc:
                 self.log.error("Exception on attempting to check workload comparison results:\n%s", exc)
                 return super().get_test_status()
+        else:
+            return super().get_test_status()
+
+    def _throughput_latency_parallel_run(self, read_cmds, test_start_time, latency_99_for_latency_workload,
+                                         latency_user, throughput_user, throughput_cmd_name, latency_cmd_name):
+        def __get_stat_for_user(read, user_name):
+            # This is handle case when both loads (latency and throughput) are run for the same user
+            stat_rate, _ = self.one_run_c_s_stats(read_run=read, user_name=user_name,
+                                                  statistic_name='latency 99th percentile')
+
+            if stat_rate:
+                latency_99_for_mixed_workload[user_name] = float(stat_rate)
+
+        self.log.debug('Start latency workload (user %s) in parallel with throughput workload '
+                       '(user %s)', latency_user, throughput_user)
+        read_queue = self.run_stress_and_verify_threads(params={'stress_cmd': [read_cmds[throughput_cmd_name],
+                                                                               read_cmds[latency_cmd_name]],
+                                                                'round_robin': True})
+
+        latency_99_for_mixed_workload = {}
+
+        # Get stats for throughput user load
+        __get_stat_for_user(read=read_queue[0], user_name=throughput_cmd_name)
+
+        # Get stats for latency user load
+        __get_stat_for_user(read=read_queue[1], user_name=latency_cmd_name)
+
+        self.assertTrue(latency_99_for_mixed_workload, msg='Not received cassandra-stress for mixed workload')
+
+        grafana_dataset = self.monitors.get_grafana_screenshot_and_snapshot(test_start_time=test_start_time)
+
+        grafana_screenshots = grafana_dataset.get('screenshots', [])
+        grafana_snapshots = grafana_dataset.get('snapshots', [])
+
+        self.log.debug('GRAFANA SCREENSHOTS: {}'.format(grafana_screenshots))
+        self.log.debug('GRAFANA SNAPSHOTS: {}'.format(grafana_snapshots))
+
+        # Compare latency of two runs
+        self.log.debug('Test results:\n---------------------\n')
+        latency_99_latency_workload = latency_99_for_latency_workload[latency_user['role'].name]
+        latency_99_mixed_workload = latency_99_for_mixed_workload[latency_cmd_name]
+        deviation = self.calculate_deviation(latency_99_latency_workload, latency_99_mixed_workload)
+        latency_change = 'increased' if latency_99_mixed_workload > latency_99_latency_workload else 'decreased'
```

Comment:
What about `latency_99_mixed_workload == latency_99_latency_workload` ?

Reply:
Good catch. Despite with the latency numbers it does not happen.
Added `latency_change = 'not changed'`

Reply:
Yeah, it's very much an edge case, but good to have it covered :)